@dgrammatiko The js linter complains about missing semicolon: https://ci.joomla.org/joomla/joomla-cms/46304/1/21

```
/********/src/build/media_source/com_media/js/edit-images.es6.js
  145:9  error  Missing semicolon  semi
```

--- a/build/media_source/com_media/js/edit-images.es6.js
+++ b/build/media_source/com_media/js/edit-images.es6.js
@@ -142,7 +142,7 @@ class Edit {
       requestAnimationFrame(() => {
         this.imagePreview.setAttribute('width', this.imagePreview.naturalWidth);
         this.imagePreview.setAttribute('height', this.imagePreview.naturalHeight);
-      })
+      });
     });
   }
 

